code/game/objects/structures/signs.dm

```diff
@@ -557,3 +557,8 @@
 	name = "\improper TGMC poster"
 	desc = "An old recruitment poster for the TGMC. Looking at it floods you with a mixture of pride and sincere regret."
 	icon_state = "prop3"
+
+/obj/structure/sign/prop4
+	name = "TerraGov: United for Humanity"
+	desc = "A poster depicting TerraGov's logo and motto, reminding marines of who's looking out for humankind against the Xenomorph menace. Semper Fi."
```

Comment:
what semper fi whoo did you get this from

Reply:
"According to marines, the term semper fi is a Latin phrase that is short for “semper fidelis.” This phrase means “always faithful.” Semper fi is the motto of every member of the U.S. Marine Corps. The United States Marine Corps uses the phrase to reflect a collective and eternal commitments to succeeding in battles, the [progress of the nation of the United States, and a loyalty to the Marine Corps at large. This phrase develops a bond between Marines and shows that being in the Marine Corps is not just a job, but a warriorhood. This phrase united individual marines and military units with their fellow Marines also performing military service, or who have served in the past."

Sprite is from TG Station.

Reply:
I can read wikipedia too but this is tgmc not the us marines

Reply:
SAD